fabric-fluid-v1/build.gradle

```diff
@@ -0,0 +1,6 @@
+archivesBaseName = "fabric-fluid-v1"
```

Comment:
This doesn't follow the convention for fabric api modules (`fabric-xyz-api-v1` unless it's some special case like `fabric-xyz-events-v1`)

Reply:
Fixed, just missing...